### DISCUSSION TITLE
milvus object storage i/o performance

### DISCUSSION THREAD

**Discussion by rere950303** [milvus-io/milvus#37362] [Q&A and General discussion] [answered]
When using object storage such as `minio`, `s3` in milvus, I think the I/O performance is not that important. Is that right?

Both memory index and disk index are cached in the querynode memory or local disk, and in the case of the growing segment, the querynode maintains it in memory and the datanode flushes it to the object storage, so the I/O performance of the object storage does not seem that important. If I am wrong, please comment.

**Comment by yhmo** [comment, marked as answer]
Minio/S3 I/O only affects the time cost for load process, doesn't affect search/query performance because data has been loaded into memory.
